doc/api/cli.md

```diff
@@ -1059,8 +1059,13 @@ Track heap object allocations for heap snapshots.
 ### `--unhandled-rejections=mode`
 <!-- YAML
 added:
- - v12.0.0
- - v10.17.0
+  - v12.0.0
+  - v10.17.0
+changes:
+  - version: v15.0.0
+    pr-url: https://github.com/nodejs/node/pull/33021
+    description: Changed default mode from warning plus a deprecation warning
+                 to `throw`.
```

Comment:
I was mildly confused for a moment by "warning plus a deprecation warning". (I know that text is in the original doc.) The deprecation warning was only emitted in some cases. (The original text is "plus a deprecation warning for the very first unhandled rejection in case no unhandledRejection hook is used.") I think that's too much detail for this and it will be more clear if we just say it was changed to throw.

```suggestion
    description: Changed default mode to `throw`. Previously, a warning was
                emitted.
```

Reply:
thx and applied suggestion.  yea its confusingly worded, and laziness on my part to figure out something clearer (instead of copy-pasting original).